ui/src/components/VolumeListTable.js

```diff
@@ -357,68 +334,48 @@ function Table({
                     </TableHeader>
                   );
                 })}
-              </HeadRow>
+              </div>
             );
           })}
-        </thead>
+        </div>
         <Body {...getTableBodyProps()}>
           {data.length === 0 ? (
-            <HeadRow
+            <div
               style={{
                 width: '100%',
                 paddingTop: padding.base,
                 height: '60px',
               }}
+              className="tr"
             >
-              <td
+              <div
                 style={{
                   textAlign: 'center',
                   background: theme.brand.primary,
+                  border: 'none',
                 }}
+                className="td"
               >
-                {intl.translate('no_volume_found')}
-              </td>
-            </HeadRow>
+                No Volume
```

Comment:
A translation seems to be missing here, maybe `{intl.translate('no_volume_found')}`